src/hooks/useServiceClient.js

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'react'
+import ServiceClient from 'lib/collab/stack-client'
+
+function useServiceClient({ userId, userName, cozyClient }) {
+  return useMemo(
+    () => {
+      return new ServiceClient({ userId, userName, cozyClient })
```

Comment:
I find ServiceClient very generic, it's hard to know what are its responsibilities from here.

Reply:
Ok about that, but it should probably be a separate issue.

-> https://github.com/cozy/cozy-notes/issues/64